README.md

```diff
@@ -2,3 +2,24 @@
 
 [![Build Status](https://travis-ci.org/dremio-hub/dremio-hbase-connector.svg?branch=master)](https://travis-ci.org/dremio-hub/dremio-hbase-connector)
 
+## Usage Instructions
+
+* From the project root run `mvn clean install`. The hbase connector jar should now be present
```

Comment:
Consider moving system responses or step results to the line immediately following the action required of the user.  For example:
1. From the project root, run ...
   The HBase connector appears in the `target` directory.

Separately, replace 'hbase' with 'HBase' to be consistent with the product name. Also, consider tightening the language a bit by replacing 'should now be present' with 'appears'.

Reply:
Done